app/routes/declaration.js

```diff
@@ -60,6 +60,11 @@ module.exports = [{
         return h.view('offer-rejected', { ruralPaymentsAgency })
       }
 
+      if (!applicationReference) {
+        console.log('Apply declaration returned a null application reference.')
+        throw boom.internal()
```

Comment:
(me being pedantic) - can we add a todo code comment to remind of that we need a proper error screen please?

In the response from the application service I dont suppose we get an error to log?

Reply:
No error is returned, all we get is what is logged to the apply log (See below)

Apply log ->  Received response {"applicationState":"failed","applicationReference":null} from queue ffc-ahwr-application-response for sessionID df5dafe6-2b04-4def-9426-82ae22776872.